client/src/components/imagine25/components/TeammateVideo.js

```diff
@@ -40,14 +42,21 @@ const TeammateVideo = (props) => {
       <p>
         Teammate live from: <b>Buffalo, NY</b>
       </p>
-      <div className="tw-pt-[100px] tw-pb-[200px]">
-        <h1>Instructions:</h1>
-        <br></br>
-        <p>Use ← → arrow keys to move</p>
-        <p>Space bar to shoot</p>
-        <p>Don&#39;t touch enemies</p>
-        <br></br>
-        <p>Get a higher score than your opponents to win!</p>
+      <div className="tw-pt-8 tw-pb-[200px] tw-grid tw-grid-cols-2 tw-grid-rows-4 tw-gap-y-6">
+        <h4 className="tw-col-span-2 tw-text-center">Movement:</h4>
+        <img
+          className={"tw-rotate-180 " + buttonSize}
+          src="/img/imagine_game_controls/ArrowKey.png"
+        />
+        <img
+          className={buttonSize}
+          src="/img/imagine_game_controls/ArrowKey.png"
+        />
```

Comment:
Let's have arrows pointing in different directions

![Screenshot 2025-03-20 at 11 40 54 AM](https://github.com/user-attachments/assets/4714d0bb-fb6a-46d8-a4a6-32619a8d9b0f)